### PR TITLE
Discord auth

### PR DESCRIPTION
**Pull request by Osmose**
🙏

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,9 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React + TS</title>
+    <title>ribbet-recipes</title>
```
Comment (ElusiveEllie) [inline index.html]: Misspelled "Ribbit" again

**Review by ElusiveEllie — CHANGES_REQUESTED** [review]
Small spelling error

File: src/components/App.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      setCurrentUser(user);
+    }
+    initAuth();
+  }, [setCurrentUser]);
+
+  return (
+    <div id="app-container">
+      <Header />
+      <Switch>
+        <Route component={Landing} />
```
Comment (ElusiveEllie) [inline src/components/App.tsx]: I think eventually we'll want the default to be a 404 page, and just have `"/"` be the landing, right?

**Comment by Osmose** [comment]
Updated with fixes

**Review by ElusiveEllie — APPROVED** [review]

**Pull request merged**